src/main/java/net/imagej/ops/filter/FilterNamespace.java

```diff
@@ -622,7 +624,7 @@ void correlate(final RandomAccessibleInterval<I> raiExtendedInput,
 			raiExtendedInput, raiExtendedKernel, fftInput, fftKernel, output,
 			performInputFFT, performKernelFFT);
 	}
-
+	
```

Comment:
According to the [coding style](http://imagej.net/Coding_style) empty lines should be empty, i.e. there shouldn't be any whitespaces (tabs, spaces, ...).